SmartDeviceLinkService.m

```diff
@@ -593,7 +584,7 @@ - (void)closeListInfoType:(MWInfoType)infoType {
     [self setCurrentInfoTypeList:nil];
     self.currentInfoTypeListIndex = -1;
     
-    [self sendWelcomeMessageWithSpeak:NO];
+    [self sendWeatherConditions:[WeatherDataManager sharedManager].weatherConditions withSpeak:YES firstWindow:NO];
```

Comment:
I'm not sure that this change is correct. It used to be welcome with speak NO, now it's current (which is correct) but with speak YES. Shouldn't that be NO?